src/vm/analysis/arithmetic_checker/tests.rs

```diff
@@ -112,7 +112,7 @@ fn test_variables_fail_arithmetic_check() {
 
     for (contract, error) in tests.iter() {
         assert_eq!(
-            arithmetic_check(contract),
+            arithmetic_check(contract, ClarityVersion::Clarity2),
```

Comment:
Can you actually test both Clarity versions here?  I don't want to lose coverage for `Clarity1`.

Reply:
Good idea.. I added tests for both versions.